src/coverlet.core/Helpers/FileSystem.cs

```diff
@@ -3,11 +3,56 @@
 
 namespace Coverlet.Core.Helpers
 {
-    internal class FileSystem : IFileSystem
+    public class FileSystem : IFileSystem
     {
         public bool Exists(string path)
         {
             return File.Exists(path);
         }
+
+        public void WriteAllText(string path, string contents)
+        {
+            File.WriteAllText(path, contents);
+        }
+
+        public string ReadAllText(string path)
+        {
+            return File.ReadAllText(path);
+        }
+
+        public Stream OpenRead(string path)
+        {
+            return File.OpenRead(path);
+        }
+
+        public void Copy(string sourceFileName, string destFileName)
```

Comment:
ditto...I don't know if it's needed.

Reply:
Sorry this was used in the ```coverlet.collector``` and I missed to revert it.